libs/sdk-ui-dashboard/src/model/store/ui/uiSelectors.ts

```diff
@@ -123,3 +123,13 @@ export const selectMenuButtonItemsVisibility = createSelector(
     selectSelf,
     (state) => state.menuButton.itemsVisibility ?? {},
 );
+
+/**
+ * @alpha
+ */
+export const selectIsInViewMode = createSelector(selectSelf, (state) => state.renderMode === "view");
```

Comment:
Maybe also a plain `selectRenderMode` could also be useful in addition to these two?